src/pageTypes/views/PageTypeList/PageTypeList.tsx

```diff
@@ -0,0 +1,176 @@
+
+  const paginationState = createPaginationState(settings.rowNumber, params);
+  const queryVariables = React.useMemo(
+    () => ({
+      ...paginationState,
+      filter: getFilterVariables(params),
+      sort: getSortQueryVariables(params)
+    }),
+    [params]
+  );
+  const { data, loading } = usePageTypeListQuery({
+    displayLoader: true,
+    variables: queryVariables
+  });
+
+  const tabs = getFilterTabs();
+
+  const currentTab =
+    params.activeTab === undefined
+      ? areFiltersApplied(params)
+        ? tabs.length + 1
+        : 0
+      : parseInt(params.activeTab, 0);
+
+  const changeFilterField = (filter: PageTypeListUrlFilters) => {
+    reset();
+    navigate(
+      pageTypeListUrl({
+        ...getActiveFilters(params),
+        ...filter,
+        activeTab: undefined
+      })
+    );
+  };
+
+  const [openModal, closeModal] = createDialogActionHandlers<
+    PageTypeListUrlDialog,
+    PageTypeListUrlQueryParams
+  >(navigate, pageTypeListUrl, params);
+
+  const handleTabChange = (tab: number) => {
+    reset();
+    navigate(
+      pageTypeListUrl({
+        activeTab: tab.toString(),
+        ...getFilterTabs()[tab - 1].data
+      })
+    );
+  };
+
+  const handleTabDelete = () => {
+    deleteFilterTab(currentTab);
+    reset();
+    navigate(pageTypeListUrl());
+  };
+
+  const handleTabSave = (data: SaveFilterTabDialogFormData) => {
+    saveFilterTab(data.name, getActiveFilters(params));
+    handleTabChange(tabs.length + 1);
+  };
+
+  const { loadNextPage, loadPreviousPage, pageInfo } = paginate(
+    data?.pageTypes?.pageInfo,
+    paginationState,
+    params
+  );
+
+  const handleSort = createSortHandler(navigate, pageTypeListUrl, params);
+
+  return (
+    <>
+      <PageTypeListPage
+        currentTab={currentTab}
+        initialSearch={params.query || ""}
+        onSearchChange={query => changeFilterField({ query })}
+        onAll={() => navigate(pageTypeListUrl())}
+        onTabChange={handleTabChange}
+        onTabDelete={() => openModal("delete-search")}
+        onTabSave={() => openModal("save-search")}
+        tabs={tabs.map(tab => tab.name)}
+        disabled={loading}
+        pageTypes={data?.pageTypes?.edges?.map(edge => edge.node)}
+        pageInfo={pageInfo}
+        onAdd={() => navigate(pageTypeAddUrl)}
+        onBack={() => navigate(configurationMenuUrl)}
+        onNextPage={loadNextPage}
+        onPreviousPage={loadPreviousPage}
+        onRowClick={id => () => navigate(pageTypeUrl(id))}
+        onSort={handleSort}
+        isChecked={isSelected}
+        selected={listElements.length}
+        sort={getSortParams(params)}
+        toggle={toggle}
+        toggleAll={toggleAll}
+        toolbar={
+          <IconButton
+            color="primary"
+            onClick={() =>
+              openModal("remove", {
+                ids: listElements
+              })
+            }
+          >
+            <DeleteIcon />
+          </IconButton>
```

Comment:
I don't see corresponding dialog window, will it be added later?

Reply:
yep, I'll add it in another pull request